python/taichi/lang/__init__.py

```diff
@@ -61,16 +61,42 @@
 
 cfg = default_cfg()
 x86_64 = _ti_core.x64
+"""The X64 CPU backend.
+"""
 x64 = _ti_core.x64
```

Comment:
A short question: why do we have two names for _ti_core.x64 ?

Reply:
likely a technical debt...